common/src/main/java/com/wynntils/models/activities/markers/ActivityMarkerProvider.java

```diff
@@ -10,59 +10,86 @@
 import com.wynntils.models.marker.type.MarkerInfo;
 import com.wynntils.models.marker.type.MarkerProvider;
 import com.wynntils.models.marker.type.StaticLocationSupplier;
+import com.wynntils.services.map.pois.MarkerPoi;
 import com.wynntils.utils.colors.CommonColors;
 import com.wynntils.utils.mc.type.Location;
+import com.wynntils.utils.mc.type.PoiLocation;
 import com.wynntils.utils.render.Texture;
-import com.wynntils.utils.type.Pair;
 import java.util.stream.Stream;
 
-public class ActivityMarkerProvider implements MarkerProvider {
-    private Pair<Location, MarkerInfo> spawnInfo;
-    private Pair<Location, MarkerInfo> trackedActivityInfo;
+public class ActivityMarkerProvider implements MarkerProvider<MarkerPoi> {
+    private static final String ACTIVITY_LOCATION_NAME = "Activity Location";
+
+    private ActivityMarkerInfo spawnInfo;
+    private ActivityMarkerInfo trackedActivityInfo;
 
     public void setSpawnLocation(Location spawnLocation) {
         this.spawnInfo = spawnLocation == null
                 ? null
-                : Pair.of(
+                : new ActivityMarkerInfo(
                         spawnLocation,
                         new MarkerInfo(
+                                ACTIVITY_LOCATION_NAME,
                                 new StaticLocationSupplier(spawnLocation),
                                 Texture.QUESTS_BUTTON,
                                 CommonColors.WHITE,
                                 CommonColors.WHITE,
-                                CommonColors.WHITE));
+                                CommonColors.WHITE),
+                        new MarkerPoi(
+                                PoiLocation.fromLocation(spawnLocation),
+                                ACTIVITY_LOCATION_NAME,
+                                Texture.QUESTS_BUTTON));
```

Comment:
I think we should either keep the "pin" icon, or make an item that fits the current content type (e.g. a cave icon for caves, etc).

Reply:
Which is kind of feature creep for this PR. So basically I am suggesting that we stay with the current pin icon.

Reply:
But I didn't change the texture?

Reply:
You didn't? The activity tracker will show up as a Quest icon in current main? I missed that part when reviewing, and I have not been playing with it...

Reply:
Yes. It is like that, it looks okay, although it could use a white border.

Reply:
So, I assume this is fine? @magicus

Reply:
Yeah, I guess :)